assets/js/components/notifications/GoogleTagIDMismatchNotification.js

```diff
@@ -124,11 +124,11 @@ export default function GoogleTagIDMismatchNotification() {
 		setHasMismatchedGoogleTagID,
 	} = useDispatch( MODULES_ANALYTICS_4 );
 
-	const updateToNewAnalyticsConfig = useCallback( () => {
-		setPropertyID( newAnalyticsConfig?.propertyID );
-		setWebDataStreamID( newAnalyticsConfig?.webDataStreamID );
-		setMeasurementID( newAnalyticsConfig?.measurementID );
-		submitChanges();
+	const updateToNewAnalyticsConfig = useCallback( async () => {
+		await setPropertyID( newAnalyticsConfig?.propertyID );
+		await setWebDataStreamID( newAnalyticsConfig?.webDataStreamID );
+		await setMeasurementID( newAnalyticsConfig?.measurementID );
```

Comment:
These setters are just regular actions, I don't think we need to `await` them. AFAIK we don't `await` any of the generated setting setters like this...